1-org/envs/shared/log_sinks.tf

```diff
@@ -34,88 +35,65 @@ resource "random_string" "suffix" {
   special = false
 }
 
-/******************************************
+module "logs_export" {
+  source = "../../modules/centralized-logging"
+
+  resources                      = local.parent_resources
+  resource_type                  = local.parent_resource_type
+  logging_destination_project_id = module.org_audit_logs.project_id
+
+  /******************************************
   Send logs to BigQuery
 *****************************************/
-
-module "log_export_to_biqquery" {
-  source                 = "terraform-google-modules/log-export/google"
-  version                = "~> 7.3.0"
-  destination_uri        = module.bigquery_destination.destination_uri
-  filter                 = local.main_logs_filter
-  log_sink_name          = "sk-c-logging-bq"
-  parent_resource_id     = local.parent_resource_id
-  parent_resource_type   = local.parent_resource_type
-  include_children       = true
-  unique_writer_identity = true
   bigquery_options = {
-    use_partitioned_tables = true
+    logging_sink_name          = "sk-c-logging-bq"
+    logging_sink_filter        = local.main_logs_filter
+    dataset_name               = "audit_logs"
+    partitioned_tables         = "true"
```

Comment:
This seems like it could be true by default in the module.

Reply:
Done